apps/aefate/src/aefa_stores.erl

```diff
@@ -190,7 +190,7 @@ store_map_to_list(Pubkey, MapId, #store{cache = Cache} = S) ->
     ?METADATA(RawId, _, _) = get_map_meta(MapId, Meta),
     Subtree = aect_contracts_store:subtree(map_data_key(RawId), Store),
     {[ {aeb_fate_encoding:deserialize(K), aeb_fate_encoding:deserialize(V)}
-      || {K, V} <- maps:to_list(Subtree) ], S1}.
+      || {K, V} <- lists:keysort(1,maps:to_list(Subtree)) ], S1}.
```

Comment:
Why not instead use a single `map_to_sorted_list/1` helper function?

Reply:
The result of this function is immediately turned back into a map with `maps:from_list` so we could refactor and let it return a map.

Reply:
Yeah, this could use more refactoring....

Reply:
Also, here we operate on map bytes => bytes. So the FATE ordering doesn't apply. This ordering doesn't rely also on any internal representation as out serialization is well defined.